docs/user/guides/generate-bindings.md

```diff
@@ -1,75 +1,82 @@
 # Generate Bindings
 
-## Requirements
+Generate a client binding for any Pulp component for the language of your choice.
 
-- Pulp 3 running on `localhost:24817`
-- Docker or Podman
+!!! warning "Language Support"
 
-## Basic Usage
+    Only python and ruby bindings have some testing, although not extensive.
 
+    For python clients, we recommend looking at [pulp-glue] as an alternative.
+    That library is handcrafted, handles tasks for you and knows how to deal with different versions of Pulp and plugins.
 
-The `generate.sh` script takes three positional arguments:
+    If you need to tweak the generator to make it work for your language, have a look at [how it works].
 
-```
-./generate.sh <module-name> <language> [<version>]  
-```
+## Overview
 
-When the optional version parameter is provided, it is used as the version string.
-When it is not provided, the version reported by Pulp's status API is used.
+- Requirements:
+    - Docker/Podman compose
+- General workflow:
+    1. Clone `pulp-openapi-generator`.
+    1. Get the openapi schema for the desired Pulp components.
+    1. Run the generator cli from the the repository
 
-The final packages are stored in the `{plugin_label}-client` directory.
+## 1) Setup
 
-Examples:
+To get started, first clone the repo into your machine.
+It'll be used as the workspace for the bindings generation.
 
 ```bash
-# Create Python Package in 'pulpcore-client/'.
-# Uses API reported  version
-sudo ./generate.sh pulpcore python
+git clone https://github.com/pulp/pulp-openapi-generator
+cd pulp-openapi-generator
+```
 
-# Create Ruby Gem in 'pulp_rpm-client/'.
-# Uses API reported version
-sudo ./generate.sh pulp_rpm ruby
+## 2) Get the schema
```

Comment:
Is it wise to number headlines?
In this very case, we use numbers 1 and 2 for another purpose already...